contributors/devel/sig-scalability/kubemark-setup-guide.md

```diff
@@ -0,0 +1,71 @@
+## Introduction
+This document serves to understand how to set up kubemark cluster given that a base cluster (to run hollow-node pods) and separate master instance(s) (to act as master(s) for the hollow nodes) are already present.
+
+## Precondition
+You need two k8s clusters(kubemark cluster and external cluster) to set up a final kubemark cluster.
+
+The functions of the two clusters are as follows:
+
+- kubemark cluster: the cluster only needs masters, its masters will be the final kubemark cluster's masters
+- external cluster: it's used to create hollow nodes for the final kubemark cluster.
+
+## Steps:
+1. Build kubemark image
+
+If the latest image of `staging-k8s.gcr.io/kubemark` in the public repo is available, you can skip this section. 
```

Comment:
I don't think "if the lastest image is available in public repo" makes sense, since we know that it's always available. Maybe instead say that "if you want to build/use your own custom kubemark image follow these steps. Otherwise skip this section and just use the latest image from public repo..."

Reply:
Done